### DISCUSSION TITLE
✔ Nuke down the passive income on MvM MA servers from 500 to 100 in the same interval

### DISCUSSION THREAD

**Discussion by leeroytm** [TitanTF/TitanTF#40] [General]
Suggestion by: masterfusion#7612 <@267283810746368001>
Located: https://discord.com/channels/774245035615977473/845235964712583208/852489327447769118

---
### Discussion happened:
>Green 
 they'll idle even longer for full money

> Jun
@ masterfusion How about......making a server without money regen
Might that help

> masterfusion — 06/10/2021
Some money regen is good but when it's THIS much then people don't bother playing normally

> Jun — 06/10/2021
I'm thinking that might prompt players to wait even longer......
If they're used to playing with so much money, then they suddenly lose all that
As a result.....conflict between players who want money and players who want to start
I say at most.......one or two MA servers, this cannot apply to all servers as it goes, somewhat, against the purpose of MA
Which is designed for people to fuck around
That's why we technically don't punish people for spamming weapons
And the best we can do.....is to punish those who "intentionally" try and crash the server through weapon spamming

> Majin Devil — 06/10/2021
So weapon restriction is a good idea then, I've already mentioned one weapon but there are other weapons too

**Comment by jiaweing** [comment]
This one is resolved with the new money regen toggle vote.
